asm2vec/tensors.py

```diff
@@ -0,0 +1,69 @@
+import os
+import torch
+import logging
+import asm2vec
+from asm2vec import utils
+from pathlib import Path
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+
+def calc_tensors(asm_path, tensor_path, model_path, epochs, device='cpu', lr=0.02) -> list:
+    """Calculates vector representation of a binary as the mean per column
+    of the vector representations of its assembly functions
+    :param asm_path: folder with assembly function in a subfolder per binary
+    :param tensor_path: folder to store the tensors
+    :param model_path: path to the trained model
+    :param epochs: number of epochs
+    :param device:  'auto' | 'cuda' | 'cpu'
+    :param lr: learning rate
+    """
+    tensors_list = []
+    if device == 'auto':
+        device = 'cuda' if torch.cuda.is_available() else 'cpu'
+
+    if os.path.isfile(model_path):
+        model, tokens = asm2vec.utils.load_model(model_path, device=device)
```

Comment:
Just use the utils you imported
```suggestion
        model, tokens = utils.load_model(model_path, device=device)
```